analog_clock/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -148,7 +128,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastUpgradeCheck = 1020;
-				ORGANIZATIONNAME = "The Chromium Authors";
+				ORGANIZATIONNAME = "";
```

Comment:
Shouldn't this be set to "The Flutter Authors" or something?

Reply:
This is the unadulterated output of `flutter create`, and it has been like this for a couple of stable releases IIRC.

Reply:
OK, no worries, then.